src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -954,6 +948,8 @@ const MeasureGroups = (props: MeasureGroupProps) => {
                           const fieldProps = {
                             name: `populations[${index}].definition`,
                           };
+                          const descriptionName = `populations[${index}].description`;
+
                           // if the population is exclusion of something then it should be in col 2
                           const isExclusionPop = population.name
                             .toLowerCase()
```

Comment:
Maybe new task but I think this can be removed now. we do not have two column layout for populations any more.